.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: e2e
+
+on:
+  pull_request:
+    branches:
+      - main
+      - release-*
+    types:
+      - labeled
+      - opened
+      - synchronize
+      - closed
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'ok-to-test') || github.event_name == 'closed' }}
```

Comment:
Does `|| closed` mean that this will trigger even if the PR is cancelled?

Reply:
Umm let me check the docs

Reply:
Removed